tests/test_mock_controller.py

```diff
@@ -51,103 +53,511 @@ async def write(self, st):
+
+        # Give some time to the mock device to move.
+        await asyncio.sleep(1)
         await self.write("status:\n")
         self.data = await self.read()
-        sau.assertReply(
-            "ApCS", self.data, status="Moving to elevation 10.0", positionActual=5
+        lwscs_status = self.data[Dome.LlcName.LWSCS.value]
+        self.assertEqual(
+            lwscs_status["status"], Dome.LlcStatus.MOVING.value,
+        )
+        self.assertGreaterEqual(
+            lwscs_status["positionActual"], 1.5,
         )
+        self.assertLessEqual(
+            lwscs_status["positionActual"], 2.0,
+        )
+
         await self.write("stopEl:\n")
         self.data = await self.read()
-        sau.assertReply("OK", self.data, Timeout=2)
+        self.assertEqual(self.data["OK"]["Timeout"], self.mock_ctrl.short_timeout)
+
+        # Give some time to the mock device to stop moving.
+        await asyncio.sleep(0.1)
         await self.write("status:\n")
         self.data = await self.read()
-        sau.assertReply("ApCS", self.data, status="Stopped", positionActual=5)
+        lwscs_status = self.data[Dome.LlcName.LWSCS.value]
+        self.assertEqual(
+            lwscs_status["status"], Dome.LlcStatus.STOPPED.value,
+        )
+        self.assertGreaterEqual(
+            lwscs_status["positionActual"], 1.5,
+        )
+        self.assertLessEqual(
+            lwscs_status["positionActual"], 2.0,
+        )
+
+    async def test_stop(self):
+        await self.write("moveAz:\n azimuth: 10\n")
+        self.data = await self.read()
+        self.assertEqual(self.data["OK"]["Timeout"], self.mock_ctrl.long_timeout)
+        await self.write("moveEl:\n elevation: 5\n")
+        self.data = await self.read()
+        self.assertEqual(self.data["OK"]["Timeout"], self.mock_ctrl.long_timeout)
+        await self.write("setLouver:\n id: 5\n position: 90.0\n")
+        self.data = await self.read()
+        self.assertEqual(self.data["OK"]["Timeout"], self.mock_ctrl.long_timeout)
+        await self.write("openShutter:\n")
+        self.data = await self.read()
+        self.assertEqual(self.data["OK"]["Timeout"], self.mock_ctrl.long_timeout)
+        await self.write("stop:\n")
+        self.data = await self.read()
+        self.assertEqual(self.data["OK"]["Timeout"], self.mock_ctrl.long_timeout)
+        # Give some time to the mock devices to move.
+        await asyncio.sleep(1)
+        await self.write("status:\n")
+        self.data = await self.read()
+        status = self.data[Dome.LlcName.AMCS.value]
+        self.assertEqual(
+            status["status"], Dome.LlcStatus.STOPPED.value,
+        )
+        status = self.data[Dome.LlcName.APSCS.value]
+        self.assertEqual(
+            status["status"], Dome.LlcStatus.STOPPED.value,
+        )
+        status = self.data[Dome.LlcName.LCS.value]
+        self.assertEqual(
+            status["status"], [Dome.LlcStatus.STOPPED.value] * 34,
```

Comment:
Named constants in the mock module (or similar) would make this kind of code a bit easier to understand.

Reply:
Done.